src/components/modals/InviteUsers.vue

```diff
@@ -139,9 +139,9 @@ export default {
           await $toasted.error(t('inviteTeammates.provide_valid_email'));
           return;
         }
-        const emails = tags.map((value) => value.text);
+        const emailsGroup = tags.map((value) => value.text);
```

Comment:
Does renaming this variable fix the issue because `emails` is already defined in the global scope?

Reply:
yes